src/components/molecules/ScheduleEventListNG/ScheduleEventListNG.tsx

```diff
@@ -51,17 +52,20 @@ export const ScheduleEventListNG: React.FC<ScheduleEventListNGProps> = ({
       <div className="ScheduleEventListNG">
         <ScheduleEventSubListNG
           events={daysEvents}
-          title={`Events on ${formatDateRelativeToNow(scheduleDate)}`}
+          title={`Coming up ${formatDateRelativeToNow(scheduleDate, {
+            formatTomorrow: () => "tomorrow",
+            formatOtherDate: (date) => "on " + format(date, "do MMM yyyy"),
```

Comment:
might as well move this `format(date, "do MMM yyyy")` into some util function that accepts `date` as an argument

Reply:
not sure why should we wrap library functions into ours. I guess it's more obvious to use it the way it's now. Moreover, there is no extra meaning: just a needed format.

But if you insist, of course, I can do that.